Unable to create a coverage report

I'm following this guide https://redhatinsights.github.io/insights-results-aggregator/testing.html#coverage-reports and run `make-coverage.sh integration` then when press `ctrl + c` I get such error:

```
2:31PM INF Producer shutting down. package=sarama
exiting...
panic: send on closed channel
goroutine 33 [running]:
github.com/Shopify/sarama.(*asyncProducer).shutdown(0xc000360380)
        /home/dmisharo/go/pkg/mod/github.com/!shopify/sarama@v1.26.0/async_producer.go:1017 +0xd9
github.com/Shopify/sarama.withRecover(0xc000079220)
        /home/dmisharo/go/pkg/mod/github.com/!shopify/sarama@v1.26.0/utils.go:45 +0x49
created by github.com/Shopify/sarama.(*asyncProducer).AsyncClose
        /home/dmisharo/go/pkg/mod/github.com/!shopify/sarama@v1.26.0/async_producer.go:297 +0x6b
FAIL    github.com/RedHatInsights/insights-results-aggregator   11.346s
total:  (statements)    0.0%
```

File `coverage.out` is empty:

```
cat coverage.out

mode: set
```

--- a/aggregator.go
+++ b/aggregator.go
@@ -457,12 +457,7 @@ func handleCommand(command string) int {
 	case "start-service":
 		printVersionInfo()
 
-		errCode := startService()
-		if errCode != ExitStatusOK {
-			return errCode
-		}
-
-		return stopService()
+		return startService()
 	case "help", "print-help":
 		return printHelp()
 	case "print-config":

